### PR TITLE
core: export signal events

### PR DESCRIPTION
**Pull request by kevinwhereby**
-------

### Description

**Summary:**
export signal events from core so we can subscribe to them in the captioner worker

**Related Issue:**
https://linear.app/whereby/issue/COB-695/clean-up-deepgram-sockets-when-participants-leave

### Checklist

-   [ ] My code follows the project's coding standards.
-   [ ] Prefixed the PR title and commit messages with the service or package name
-   [ ] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [ ] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/core/src/redux/slices/signalConnection/index.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ import {
 import { Credentials } from "../../../api";
 import { selectAppIsActive } from "../app";
 import { signalEvents } from "./actions";
+export { signalEvents } from "./actions";
```
Comment (nandito) [inline packages/core/src/redux/slices/signalConnection/index.ts:32]: Wouldn't it look better if this is at the bottom of the file? And as this is already imported, maybe the `from "./actions"` is not necessary.

Comment (kevinwhereby) [reply packages/core/src/redux/slices/signalConnection/index.ts:32]: Hmmm TS doesn't like any other way I've found to export it, besides:
```typescript
```

Comment (kevinwhereby) [reply packages/core/src/redux/slices/signalConnection/index.ts:32]: happy to move it to the bottom if you prefer though, personally I like keeping the two together

**Review by nandito — APPROVED** [review]
LGTM 🍰

**Pull request merged**